test/unit/Sensorario/DiamondKata/DiamondKataTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Sensorario\DiamondKata;
+
+use PHPUnit_Framework_TestCase;
+
+final class DiamondKataTest extends PHPUnit_Framework_TestCase
+{
+    public function testOneDimentionalMatrix()
+    {
+        $matrix = DiamondKata::createFrom('A');
```

Comment:
Mmmm Diamond::withChar('A'); is better than DiamondKata::createFrom('A'); I dont think is created any kata here ajajjaaj. I think a Diamond is. So, ... What do you think about ...

    Diamond::withChar('A');

or

    Matrix::startingFrom('A');
    Diamond::with('A');
    Matrix::withChar('A');

Ok, ... the purpose here is to share opinions. I am inviting you to investigate a strategy to create class/method names, so the next pair/kata session we can code better XD.